packages/contracts-bedrock/src/L2/L2CrossDomainMessenger.sol

```diff
@@ -66,13 +70,34 @@ contract L2CrossDomainMessenger is CrossDomainMessenger, ISemver {
         override
         returns (bool)
     {
+        // TODO: Remove this comment 5_500 accounted for in this call.
```

Comment:
need to include the check for if it's a forced L1 -> L2 message. if it's not a forced message then we skip everything else